workers/unity/Packages/com.improbable.gdk.core/Subscriptions/EntityGameObjectLinker.cs

```diff
@@ -127,6 +99,11 @@ public EntityGameObjectLinker(World world)
             }
 
             gameObjectToInjectors.Add(gameObject, injectors);
+
+            if (entityId.IsValid())
```

Comment:
Why is this conditional on the EntityId being valid? 

Is there a reason to prevent GameObject components being pushed onto the worker entity?

Reply:
Ah I see - its to stop the exception from being thrown.

Reply:
Is this a regression on the core v1 implementation?